src/components/editableField/EditableField.js

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react';
+import './editableField.scss';
+
+export class EditableField extends Component {
+  state = {
+    isHidden: true,
+    value: 'input value...',
```

Comment:
As a customer of your component I don't want to see `input value...`. It should be an empty, or give me a possibility to set this default text by myself (via props)